tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -330,7 +330,7 @@ testParameters = {
 testConnection : connection
 testConnection = {
             handshake = {handshake_type = 0x00000000,
-                message_number = 0x00000000},
+                message_number = 0x00000000, hello_retry_request = 0x00},
```

Comment:
Can this go on its own line to match the format of the rest of the connection?

Reply:
Updated